src/main/java/subway/line/LineService.java

```diff
@@ -0,0 +1,61 @@
+package subway.line;
+
+import org.springframework.stereotype.Service;
+import subway.station.StationQuery;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class LineService {
```

Comment:
소소한 성능개선을 할 수 있을 것 같아요 😄 
```suggestion
@Service
Transactional(readOnly = true)
public class LineService {
```

Reply:
저도 우테코 과정에서는 항상 readOnly를 붙였었는데요, 실무에서 사용해보니 one source를 사용하는 경우 readOnly=true 옵션을 주게되면 아래와 같이 불필요하게 CPU에 부하를 주는 empty transaction이 발생한다고 해요. 만약 readOnly를 붙이지 않으면 별도의 트랜잭션 없이 조회가 이루어지구요. 그래서 습관적으로 안붙이게 됐던 것 같아요!!
```
SET AUTOCOMMIT=OFF;
SET SESSOIN TRANSACTION READ ONLY [WRITE];
-- Nothing --
COMMIT;
```
하지만 이번에는 베스트 프랙티스를 위해 한번 붙여볼게요ㅎㅎ

Reply:
이 부분은 저도 한번 확인해 봐야겠네요 🤔